plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryStorageArrowPageSource.java

```diff
@@ -135,8 +139,8 @@ public Page getNextPage()
     public long getMemoryUsage()
     {
         long memoryUsage = streamBufferAllocator.getAllocatedMemory();
-        if (split.getDataSize().isPresent()) {
-            memoryUsage += split.getDataSize().getAsInt() + pageBuilder.getSizeInBytes();
+        if (splitDataSize.isPresent()) {
+            memoryUsage += splitDataSize.getAsInt() + pageBuilder.getSizeInBytes();
```

Comment:
pageBuilder.getSizeInBytes -> pageBuilder.getRetainedSizeInBytes in separate commit